docs/tensorboard_profiling_keras.ipynb

```diff
@@ -541,7 +541,7 @@
         "colab_type": "text"
       },
       "source": [
-        "Re-launch TensorBoard to observe the performance profile for the updated input pipeline. "
+        "Re-launch TensorBoard and open profile tab to observe the performance profile for the updated input pipeline. "
```

Comment:
nit: “open **the** profile tab” (missing article)

Reply:
Done, thanks.